### ISSUE TITLE
HomeKit instructions are not correct

### ISSUE THREAD

**Issue by palazzem** [palazzem/ha-econnect-alarm#141]
**Describe the bug**
HomeKit docs are not correct and the current configuration creates multiple instances. We should update it to:
```
homekit:
  - filter:
      include_domains:
        - alarm_control_panel
    entity_config:
      alarm_control_panel.alarm_panel:
       code: <PIN CODE>
    name: HASS Bridge Alarm  # Remove the YAML list block (`-`)
    port: 21065
```

**Error message**
No errors, but multiple instances are created in HomeKit and the `filter` is ignored.

**Expected behavior**
Having only one working instance.

**Additional context**
n/a

**To Reproduce**
n/a

**Environment**
 - Operating System (OS):
 - Library version (commit or version number):
 - Home Assistant version:
 - Last working Home Assistant release (if known):

**Comment by palazzem** [comment]
We should also update this section:
```yaml
    entity_config:
      alarm_control_panel.<YOUR_ALARM_PANEL_ENTITY_ID>:
       code: <PIN CODE>
```

**Comment by palazzem** [comment]
Closed with #142
